elvis/views/common.py

```diff
@@ -28,6 +28,8 @@ def determine_perms(self, request, *args, **kwargs):
 
         if not obj.__dict__.get('public', True):
             return {'can_edit': False, 'can_view': False}
+        elif obj.__dict__.get('hidden', True):
```

Comment:
This means that if 'hidden' is undefined on the dict, then the item defaults to being hidden. Is that how it's supposed to work?